### PR TITLE
Sort finished games by time

### PR DESCRIPTION
**Pull request by ghost**
Sort solved games by last played or play time -Jesus Fuentes

### CODE REVIEW

**Comment by ogarcia** [comment]
I just tested your PR, but this not is the idea. The sort option must work independently of filter. If a game has not played or not finished then the time is 00:00 (go bottom).

File: app/src/main/java/org/moire/opensudoku/gui/SudokuListSorter.java [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+        this.context = context;
+        this.sortType = sortType >= 0 && sortType < 3 ? sortType : SORT_BY_CREATED;
+    }
+
+    public int getSortType() {
+        return sortType;
+    }
+
+    public void setSortType(int sortType) {
+        this.sortType = sortType >= 0 && sortType < 3 ? sortType : SORT_BY_CREATED;;
```
Comment (ogarcia) [inline app/src/main/java/org/moire/opensudoku/gui/SudokuListSorter.java]: I think that in this line there is an extra semicolon

**Comment by fuentesj11** [comment]
Sorry I was under what the individual had asked for as for just completed or solved as that is what he asked and what was discussed. I can change this for any filter option.

**Comment by ghost** [comment]
Do you want solved puzzles to always go to the top over currently playing (which is over never started)?

File: app/src/main/java/org/moire/opensudoku/gui/SudokuListSorter.java (hunk 1 above)
Comment (ghost) [reply app/src/main/java/org/moire/opensudoku/gui/SudokuListSorter.java]: Removed. Thank you.

**Comment by ogarcia** [comment]
The current approach is better, but I think that is better revert changes in `sudoku_list.xml` and not show any of order here because these text only is showed when a filter is active
Update: And in several devices these extra info breaks view :-(

**Comment by ghost** [comment]
Removed the changes to the view at the bottom of the layout.

**Comment by ogarcia** [comment]
Merged in develop!

**Pull request closed without merging**